src/solvers/mp/rcspp/PriceLabelsGraph.h

```diff
@@ -0,0 +1,42 @@
+//
+// Created by antoine legrain on 2020-04-11.
+//
+
+#ifndef NURSESCHEDULER_PRICELABELSGRAPH_H
+#define NURSESCHEDULER_PRICELABELSGRAPH_H
+
+
+#include <vector>
+
+class SubProblem;
+
+class PriceLabelsGraph {
```

Comment:
Add comment to class?

Reply:
done in  6efc95cf3ddeb8d97e69f945b603238e901ba853